README.md

```diff
@@ -1 +1,51 @@
+# python-selenium-browserstack
 
+## Prerequisite
+```
+# If pip is not installed, you can install it using:
+sudo easy_install pip
+
+# If you prefer pip, then use the following command:
+sudo pip install selenium
+
+pip install browserstack-local
+```
+
+## Steps to run test session
+
+1. Change the capabilities if you wish:
+(For single test session, Navigate to ./scripts/single.py)
+```python
+desired_cap = {
```

Comment:
Need to pass `bstack:options` in case of Sel 4. Check [Selenium 4 Capabilities as per W3C Protocol](https://www.browserstack.com/automate/capabilities?tag=selenium-4)